test/discretization.jl

```diff
@@ -1,4 +1,13 @@
 @testset "Discretization" begin
+  function is_fully_connected(mesh::UnstructuredMesh)
+    inds = collect(1:length(vertices(mesh)))
+    connected = collect(Iterators.flatten(getproperty.(mesh.connec, :list)))
+    all(inds .== sort(unique(inds)))
+  end
+  function has_same_vertices(𝒫::PolyArea, mesh::UnstructuredMesh)
+    Set(vcat(vertices.(chains(𝒫))...)) == Set(vertices(mesh))
```

Comment:
If I remember correctly the most intuitive thing should work: `Set(vertices(P)) == Set(vertices(mesh))`

Reply:
`vertices` does not work on `PolyArea` objects:
```julia
Test threw exception
  Expression: Set(vertices(𝒫)) == Set(vertices(mesh))
  type PolyArea has no field vertices
```